packaging/command-line-assistant.spec

```diff
@@ -140,6 +140,7 @@ fi
 
 # System units
 %{_unitdir}/%{daemon_binary_name}.service
+%{_tmpfilesdir}/%{daemon_binary_name}.conf
```

Comment:
Pushed as part of https://github.com/rhel-lightspeed/command-line-assistant/pull/407